### ISSUE TITLE
knox client s3

### ISSUE THREAD

**Issue by heron2014** [DRDD2016/api#66]
Steps:

I will use https://github.com/Automattic/knox library to interact between server and S3 
 
- ```npm install knox --save``` 
- create a client 

```js
import knox from 'knox';
const knoxClient = knox.createClient({
  key: , // these values are on amazon steps below:
  secret: ,
  bucket: 
})
```

### Location of these properties on amazon:
Go back to the console, and click on your name at the top bar, then click on Security Credentials from dropdown:

<img width="193" alt="screen shot 2017-02-14 at 16 07 54" src="https://cloud.githubusercontent.com/assets/8476884/22937188/cb062f04-f2cf-11e6-961e-156df0e61776.png">

- click on ```Continue ...```
- click on ```Access Keys```
- click on ```Create new access key```

<img width="373" alt="screen shot 2017-02-14 at 16 11 34" src="https://cloud.githubusercontent.com/assets/8476884/22937376/51764286-f2d0-11e6-83a7-6670f5d3ae91.png">

- click on ```show access key```

Defined in you app the following variables (hide them!):
S3ACCESSKEY=<Access Key ID>
S3SECRET=<Secret Access key>
S3BUCKET=<your bucket name>

and pass these values to knoxClient

Then close it.

**Comment by heron2014** [comment]
### Resize the image: 
We will use ```GraphicsMagick and ImageMagick for node.js```, to be able to use this module first we need to install imagemagic too because it relies on it like it says in here: https://www.npmjs.com/package/gm

**Comment by heron2014** [comment]
Then perform next steps: 

- ```npm install gm``` this module is interacting with above libraries in node

**Comment by heron2014** [comment]
Resolved by #67
